File: app/src/main/java/com/example/requestapp/ui/fragment/finishTasktFragment/FinishTaskFragmentContract.java
package com.example.requestapp.ui.fragment.finishTasktFragment;

public interface FinishTaskFragmentContract {
    interface Presenter{

        void onClickedDoItAgain(String piority,String descryption);

    }
    interface View{

        void showMessage(String message);

        void closeFragment();

    }
}

File: app/src/main/java/com/example/requestapp/ui/fragment/finishTaskFragment/FinishTaskFragmentContract.kt
package com.example.requestapp.ui.fragment.finishTaskFragment

interface FinishTaskFragmentContract {
    interface Presenter {
        fun onClickedDoItAgain(piority: String?, descryption: String?)
    }

    interface View {
        fun showMessage(message: String?)
        fun closeFragment()
    }
}